spec/integration/operations_spec.rb

```diff
@@ -134,9 +134,20 @@ module Hawkular::Operations::RSpec
       end
 
       describe 'Operation/Operation' do
-        before(:all) do
-          @random_uuid = SecureRandom.uuid
-          @not_so_random_uuid = 'not_so_random_uuid'
+        let(:random_uuid) do
+          SecureRandom.uuid
+        end
+
+        let (:not_so_random_uuid) do
```

Comment:
Actually i need `not_so_random_uuid` to stay the same between runs. I suppose the name could be changed to `fixed_id` or something like that.